build-in-container/mariner-docker-builder.sh

```diff
@@ -76,7 +102,7 @@ while (( "$#")); do
     -i ) container_type="interactive"; shift ;;
     -c ) cleanup; exit 0 ;;
     --mariner_dir ) mariner_dir="$(realpath $2)"; shift 2 ;;
-    --RPM_repo_file ) enable_custom_repofile=true; cp $(realpath $2) $custom_repo_file; shift 2 ;;
+    --RPM_repo_file ) enable_custom_repofile=true; validate_custom_repo_file "$2"; copy_custom_repo_file "$2"; shift 2 ;;
```

Comment:
I'd suggest we move to ` --RPM_repo_files` here to better match the meaning of the flag. We are probably early enough to not worry about back-compat, but if you want to be careful you can match against both:

```suggestion
    --RPM_repo_file | --RPM_repo_files ) enable_custom_repofile=true; validate_custom_repo_file "$2"; copy_custom_repo_file "$2"; shift 2 ;;
```

Reply:
I'll omit user facing changes if they're not impacting performance. Maybe we can do a user facing change in the future and rename some of our args.